easybuild/easyconfigs/w/wheel/wheel-0.31.1-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'PythonPackage'
+
+name = 'wheel'
+version = '0.31.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/wheel'
+description = """A built-package format for Python."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['0a2e54558a0628f2145d2fc822137e322412115173e8a2ddbe1c9024338ae83c']
+
+dependencies = [('Python', '3.6.6')]
+
```

Comment:
Shouldn't this one also use
download_dep_fail = True
use_pip = True

like you told me to use for Inelastica?

Reply:
Yes, it should, added that (consistently) in ff7dbe402